### PR TITLE
Calculate and plot specificity for T-regs (#35)

### PR DESCRIPTION
**Pull request by zoeskim**

### CODE REVIEW

**Comment by zoeskim** [comment]
@aarmey ^ Should I fix all of those lines? They don't show up as Pylint errors on Jenkins (I guess the max line length is 200 there?)

**Comment by aarmey** [comment]
@zoeskim no, you can ignore this comment.

File: ckine/figures/figureS6.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+"""
+This creates Figure S6.
```
Comment (aarmey) [inline ckine/figures/figureS6.py]: This should be Fig. 6, not S6.

File: ckine/figures/figureS6.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    scales = np.zeros((4, 2))
+    for i, cells in enumerate(cell_groups):
+        subset_df = df[df['Cells'].isin(cells)]
+        scales[i, :] = optimize_scale(np.array(subset_df.loc[(subset_df["Activity Type"] == 'Predicted'), "Activity"]),
+                                      np.array(subset_df.loc[(subset_df["Activity Type"] == 'Experimental'), "RFU"]))
+
+    return scales
+
+
+def optimize_scale(model_act, exp_act):
```
Comment (aarmey) [inline ckine/figures/figureS6.py]: Import this from Fig. S5?

**Comment by aarmey** [comment]
@zoeskim why doesn't this just reuse all the code in Fig. S5? From there you get all the predictions and scaling calculated, then it's just a matter of plotting...

**Comment by zoeskim** [comment]
@aarmey I originally was reusing/editing the mutein functions because I wasn't sure if we would add them, but I will change it to use the S5 code tonight.

**Comment by zoeskim** [comment]
Fixed! One thing I noticed though is that the S5 code doesn't share scaling parameters between cell groups. Should I change that in a separate PR?

**Comment by aarmey** [comment]
Sure!

**Pull request merged**